homeassistant/components/yeelight/light.py

```diff
@@ -10,6 +10,7 @@
     Flow,
     RGBTransition,
     SleepTransition,
+    flows,
     transitions as yee_transitions,
```

Comment:
Is this still needed when using `flows`? Other than that, it looks pretty straightforward to me. As I don't actively use yeelight devices, I'd say it's @zewelor's call to merge :-)

Reply:
Good point, @rytilahti : I'll create a PR to python-yeelight to add a count parameter to the pulse function. Once that's been accepted and put into a release, I'll create a separate PR here to eliminate the dependency to yee_transitions.